src/apis/CameraRoll.re

```diff
@@ -23,43 +23,39 @@ external getPhotosParams:
   getPhotosParams =
   "";
 
-type photoIdentifier = {
-  .
-  "node": {
-    .
-    "_type": string,
-    "group_name": string,
-    "image": {
-      .
-      "filename": string,
-      "uri": string,
-      "height": float,
-      "width": float,
-      "isStored": Js.Nullable.t(bool),
-      "playableDuration": float,
-    },
-    "timestamp": float,
-    "location":
-      Js.Nullable.t({
-        .
-        "latitude": Js.Nullable.t(float),
-        "longitude": Js.Nullable.t(float),
-        "altitude": Js.Nullable.t(float),
-        "heading": Js.Nullable.t(float),
-        "speed": Js.Nullable.t(float),
-      }),
-  },
+type photoIdentifier = {node}
+and node = {
+  [@bs.as "type"]
+  _type: string,
+  group_name: string,
+  image,
+  timestamp: float,
+  location: option(location),
+}
+and image = {
+  filename: string,
+  uri: string,
+  height: float,
+  width: float,
+  isStored: option(bool),
+  playableDuration: float,
+}
+and location = {
+  latitude: option(float),
+  longitude: option(float),
+  altitude: option(float),
+  heading: option(float),
+  speed: option(float),
```

Comment:
I think we should just reorder the type definitions instead of using `and`.

Reply:
Done.